applications/osb-portal/src/pages/RepositoriesPage/RepositoriesPage.tsx

```diff
@@ -289,7 +289,7 @@ export const RepositoriesPage = ({ user }: { user: UserInfo }) => {
 
         {repositories ?
           <Box className="verticalFill" style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-between' }}>
-            <Repositories repositories={repositories} handleRepositoryClick={(repositoryId: number) => openRepoUrl(repositoryId)} refreshRepositories={() => updateList(tabValue)} />
+            <Repositories repositories={repositories} handleRepositoryClick={(repositoryId: number) => openRepoUrl(repositoryId)} refreshRepositories={() => updateList(tabValue)} handleTagClick={ (tag: Tag) => searchFilterValues.tags.indexOf(tag.tag) === -1 ? setSearchFilterValues({ ...searchFilterValues, tags: searchFilterValues.tags.concat(tag.tag)}) : console.log("Tag exists, no op.") } handleTagUnclick={ (tag: Tag) => setSearchFilterValues({ ...searchFilterValues, tags: searchFilterValues.tags.filter(t => t !== tag.tag) })} searchFilterValues={searchFilterValues} handleTypeClick={(type: string) => setSearchFilterValues({...searchFilterValues, types: searchFilterValues.types.concat(type)})} handleTypeUnclick={(type: string) => setSearchFilterValues({...searchFilterValues, types: searchFilterValues.types.filter(t => t !== type)})}/>
```

Comment:
Minor remark: this code could un-inlined to improve readability

Reply:
Yeh, I was wondering about that. I'll go break it into individual lines.